drizzle/0000_i_message_migration.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `i_message` (
+	`id` integer PRIMARY KEY NOT NULL,
+	`annoyance` text,
+	`message` text,
+	`because` text
```

Comment:
Putem pune created/updatedOn nativ? Probabil o sa avem nevoie sa le afisam by updatedOn desc aici si in alte locuri.